src/test/java/appeng/util/ConfigInventoryTest.java

```diff
@@ -77,15 +77,15 @@ class StacksMode {
         ConfigInventory inv = ConfigInventory.configStacks(1).supportedType(AEKeyType.items()).build();
 
         @Test
-        void stacksWithAmountZeroAreDropped() {
+        void stacksWithAmountZeroFilterToOne() {
             inv.setStack(0, ZERO_STICK);
-            assertNull(inv.getStack(0));
+            assertEquals(ONE_STICK, inv.getStack(0));
```

Comment:
The tests should also verify that calling `setStack` a second time does indeed remove the stack.